<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controls_widget</class>
 <widget class="QWidget" name="Controls_widget">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QPushButton" name="btn_click">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QToolButton" name="btn_clear_strings">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_add_strings">
         <property name="text">
          <string>add strings</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="cb_fag_exit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>flag exit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="rb_test">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QToolButton" name="btn_set_dt">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit">
         <property name="displayFormat">
          <string>dd.MM.yyyy HH:mm:ss</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
     <item row="1" column="1">
      <widget class="QTimeEdit" name="timeEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLongLongSpinBox" name="hex_widget"/>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="cb_checkbox_test">
       <property name="text">
        <string>CheckBox</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QDateEdit" name="de_begin"/>
       </item>
       <item>
        <widget class="QDateEdit" name="de_test"/>
       </item>
       <item>
        <widget class="QDateEdit" name="de_end"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="MyListWidget" name="listWidget"/>
     </item>
     <item>
      <widget class="QTextEdit" name="te_test">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Liberation Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyListWidget</class>
   <extends>QListWidget</extends>
   <header>mylistwidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>QLongLongSpinBox</class>
   <extends>QSpinBox</extends>
   <header>QLongLongSpinBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
